vagrant/playbooks/source-install.yml

```diff
@@ -12,11 +12,34 @@
 
     # The version string below is the highest of all those in roles' metadata:
     # "min_ansible_version". It needs to be kept manually up-to-date.
+    # FIPS boxes require 2.10
```

Comment:
Thank you, now I understand why.

Reply:
Wait, do actual user systems in FIPS mode also require 2.10?

If so, we should probably document that.

Reply:
we had this discussion a while ago (I don't remember where), but I remember we agreed that we don't officially support FIPS so we couldn't change the ansible requirement due to FIPS